src/hnsw.c

```diff
@@ -99,10 +99,13 @@ hnswcostestimate(PlannerInfo *root, IndexPath *path, double loop_count,
 {
 	GenericCosts costs;
 	int			m;
-	int			entryLevel;
+	double		entryLevel;
+	double		layer0TuplesMax;
+	double		layer0Selectivity;
+	double		scalingFactor = 0.55;
 	Relation	index;
 
-	/* Never use index without order */
+	/* Never use index without order or if there are no tuples */
```

Comment:
This comment needs reverted.

Reply:
🤦 Sorry for the sloppiness - that's due to rushing. Pushed fix.